app/controllers/admin/features_controller.rb

```diff
@@ -13,7 +13,7 @@ def create
     end
 
     if @feature.save
-      @feature.republish_organisation_to_publishing_api
+      @feature.republish_featurable_to_publishing_api
```

Comment:
So when this happens are we republishing the featured thing or the world location news page that they're featured on? I'd hope there'd be no need for the former but 'featurable' is an ambiguous name...

Reply:
The `featurable` is the page that the item is featured on (i.e. the World Location News or Organisation page).  I chose this name as it matches the database field.

Reply:
oh yeah, I've seen it before, not a complaint about your use of it, more its lack of description